File: scholar-business/src/main/java/com/varun/scholar/business/data/entities1/PerformanceSearch.java
package com.varun.scholar.business.data.entities1;

import java.util.Date;

import javax.persistence.Entity;

import com.varun.scholar.business.data.entities.AbstractEntity;

@Entity
public class PerformanceSearch extends AbstractEntity {
	String admissionNumber;
	String studentName;
	String courseName;
	String batchName;
	Date admissionDate;
	Date lastChange;
}

File: scholar-web/src/main/webapp/app/model/student/performance/PerformanceSearch.js
Ext.define('scholar.model.student.performance.PerformanceSearch', {
	extend: 'Ext.data.Model',
    fields : [ {
		name : 'id',
		type : 'int'
	}, {
		name : 'admissionNumber',
		type : 'string'
	}, {
		name : 'studentName',
		type : 'string'
	}, {
		name : 'courseName',
		type : 'string'
	}, {
		name : 'batchName',
		type : 'string'
	}, {
		name : 'admissionDate',
		type : 'date',
		dateFormat : 'n/j h:ia'
	}, {
		name : 'lastChange',
		type : 'date',
		dateFormat : 'n/j h:ia'
	} ]
	
});